lib/Common/Memory/MarkContext.inl

```diff
@@ -22,8 +22,12 @@ bool MarkContext::AddMarkedObject(void * objectAddress, size_t objectSize)
     RECYCLER_STATS_INTERLOCKED_INC(recycler, scanCount);
 
     MarkCandidate markCandidate;
-    markCandidate.obj = (void **) objectAddress;
+#if defined(_WIN64) && defined(_M_X64)
+    *(__m128i *)&markCandidate = _mm_set_epi64x(objectSize, (__int64)objectAddress);
+#else
+    markCandidate.obj = (void**)objectAddress;
     markCandidate.byteCount = objectSize;
```

Comment:
Should the bytecount be set in the #if case as well?

Reply:
I guess that intrinsic set both addr and size fields though xmm register